### PR TITLE
X-Camp USACO January Workshop Page

### PR DESCRIPTION
**Pull request by nathangong**

### CODE REVIEW

**Comment by RealMyth21** [comment]
looks good!

**Review by RealMyth21 — APPROVED** [review]

File: src/components/workshops/usaco-january/Hero.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+                  href="https://us06web.zoom.us/j/84976667940"
+                  className="underline text-blue-600"
+                >
+                  ID: 849 7666 7940
+                </a>
+              </p>
+              <div className="mt-5 sm:mt-8 sm:flex sm:justify-center lg:justify-start">
+                <div className="rounded-md shadow">
+                  <a
+                    href="https://forms.gle/X4GNptRtFAKjzynv6"
```
Comment (maggie-j-liu) [inline src/components/workshops/usaco-january/Hero.tsx:38]:
```suggestion
                    href="https://forms.gle/tZEVgL17gYAauow18"
```

File: src/components/workshops/WorkshopData.ts [hunk 2]
```diff
@@ -12,6 +12,14 @@ export type Workshop = {
 }
 
 const workshops: Workshop[] = [
+  {
+    photo: USACO,
+    title: "USACO January Contest Solutions",
+    description: "Gain an in-depth understanding of the solutions for the January Contest through a joint workshop with X-Camp!",
+    date: dayjs("Feb 5, 2022 7:00:00 PM PST"),
+    url: "usaco-january",
+    themeColor: "yellow",
```
Comment (maggie-j-liu) [inline src/components/workshops/WorkshopData.ts:21]: btw this won't always work as expected; see https://tailwindcss.com/docs/content-configuration#dynamic-class-names

Comment (nathangong) [reply src/components/workshops/WorkshopData.ts:21]: do you know of any way to solve this?

**Review by maggie-j-liu — APPROVED** [review]

**Pull request merged**